app/src/main/java/com/nervousfish/nervousfish/activities/ContactActivity.java

```diff
@@ -29,6 +33,33 @@ protected void onCreate(final Bundle savedInstanceState) {
 
         final ListView lv = (ListView) findViewById(R.id.listView);
 
+        final ImageButton backButton = (ImageButton) findViewById(R.id.backButton);
+        backButton.setOnClickListener(new View.OnClickListener() {
+            public void onClick(final View v) {
+                finish();
+            }
+        });
+
+        final ImageButton deleteButton = (ImageButton) findViewById(R.id.deleteButton);
+        deleteButton.setOnClickListener(new View.OnClickListener() {
+            public void onClick(final View v) {
+                new SweetAlertDialog(ContactActivity.this, SweetAlertDialog.WARNING_TYPE)
+                        .setTitleText("Are you sure?")
```

Comment:
Can you put these strings in strings.xml instead of hardcoding it? If I remember correctly you can use getResources().getString(R.string.foobar)